### PR TITLE
📖 Development, Nightly, and Release schedule documentation updates

### PR DESCRIPTION
**Pull request by danielrozenberg**
* Move opt-in instructions from release-schedule.md to DEVELOPING.md
* Add details about the nightly release channel to release-schedule.md
* Move description of beta/experimental channels into the weekly section of release-schedule.md and remove the opt-in instructions (links to DEVELOPING.md instead)
* Minor fixes (e.g., fixing broken links to PR labels)

### CODE REVIEW

File: contributing/release-schedule.md [hunk 1]
```diff
@@ -17,67 +18,56 @@ The AMP runtime and extensions are provided through a variety of different _rele
...
-- The [_PR Use: In Canary_](https://github.com/ampproject/amphtml/issues?utf8=%E2%9C%93&q=label%3A%22PR%20use%3A%20In%20Canary%22), [_PR Use: In Production_](https://github.com/ampproject/amphtml/issues?utf8=%E2%9C%93&q=label%3A%22PR%20use%3A%20In%20Production%22), and [_PR Use: In LTS_](https://github.com/ampproject/amphtml/issues?utf8=%E2%9C%93&q=label%3A%22PR%20use%3A%20In%20LTS%22) labels are added to PRs when they've made it into a _weekly_ or **lts** build. There may be a delay between when the build is created and when the label is added.
+The _Beta_ and _Experimental Channels_ are pre-release candidates for the next Stable release of AMP. Every Tuesday (except for releases-freeze weeks), last week's **nightly** is promoted to the developer opt-in channels for **beta** and **experimental**. Following a 1-day period where we verify that no feature or erformance regressions were introduced in these channels, we promote this release on Wednesday to a small portion of traffic. This same release is then promoted to the **stable** channel on Tuesday the following week.
```
Comment (rcebulko) [inline contributing/release-schedule.md:39]: Maybe link to the release freezes section here

Comment (rcebulko) [reply contributing/release-schedule.md:39]: `%s/erformance/performance/

File: contributing/release-schedule.md [hunk 2]
```diff
@@ -17,67 +18,56 @@ The AMP runtime and extensions are provided through a variety of different _rele
...
+### Nightly
 
-The **stable** release channel is built with the _production_ configuration and served to most AMP traffic. Since the **beta** release channel is also built from the _production_ configuration, it represents the exact build which will become **stable** the following week (modulo the possibility of cherry-picks; see [Contributing Code](https://github.com/ampproject/amphtml/blob/master/contributing/contributing-code.md#Cherry-picks)).
+The **nightly** release channel is updated (as its name indicates) every weeknight. This process is automated, and there is no guarantee that any given nightly release is free of bugs or other issues. Each night after midnight (Pacific Time), the last "green" commit from the day is selected to be the release cutoff point. A green build indicates that all automated tests have passed on that build.
 
-### Long-Term Stable (lts)
+The nightly release provides a mechanism to detect and resolve issues quickly and before they reach the more traffic-heavy _weekly_ release channels. It also serves to reduce the number of users affected by newly introduced issues
```
[the remark this answers is not in the document]
Comment (danielrozenberg) [reply contributing/release-schedule.md:25]: Done.

File: contributing/release-schedule.md (hunk 1 above)
Comment (danielrozenberg) [reply contributing/release-schedule.md]: Done + Done.

File: contributing/release-schedule.md [hunk 3]
```diff
@@ -17,67 +18,56 @@ The AMP runtime and extensions are provided through a variety of different _rele
...
 
-The **lts** release channel provides a previous **stable** build for one-month intervals. On the second Monday of each month, the current **stable** release is promoted to **lts**. This channel is not recommended for all AMP publishers. It is provided so that publishers who wish to perform a QA cycle on their website less frequently may do so by opting specific web pages into the **lts** channel (see the [**lts** readme](https://github.com/ampproject/amphtml/blob/master/contributing/lts-release.md)).
+The **nightly** release build is served to 0.05% of AMP traffic. It is possible to opt into the **nightly** channel, to test pull requests that were merged in the past few days. See the [opt-in section](DEVELOPING.md#opting-in-to-pre-release-channels) in [DEVELOPING.md] for details.
```
[the remark this answers is not in the document]
Comment (danielrozenberg) [reply contributing/release-schedule.md:27]: That sounds extremely familiar. Was the discussion about the values or about including that at all? // attn: @mrjoro

[the remark this answers is not in the document]
Comment (rcebulko) [reply contributing/release-schedule.md:27]: About including it at all

[the remark this answers is not in the document]
Comment (danielrozenberg) [reply contributing/release-schedule.md]: ok, removed that part

File: contributing/DEVELOPING.md [hunk 4]
```diff
@@ -94,4 +94,25 @@ We also recommend scanning the [spec](../spec/). The non-element part should hel
...
 - Understanding the [AMP release process](release-schedule.md) is useful for understanding when a change in AMP will make it into production and what to do if things go wrong during the rollout of a change.
 
+### Opting in to pre-release channels
+
+Developers and users of AMP can have their browser request AMP JS files from the pre-release channels (**beta**, **experimental**, and **nightly**) for all requests, using an opt-in mechanism.
+
+To opt your browser into the a pre-release channel, go to [the AMP experiments page](https://cdn.ampproject.org/experiments.html) and activate the experiment channel of your choice (see [Beta and Experimental channels](release-schedule.md#beta-and-experimental-channels) and [Nightly](release-schedule.md##nightly) in the [release process](release-schedule.md) document for description of these channels). Please subscribe to our [low-volume announcements](https://groups.google.com/forum/#!forum/amphtml-announce) mailing list to get notified about important/breaking changes about AMP.
```
Comment (rsimha) [inline contributing/DEVELOPING.md:101]: This sentence reads "see beta and experimental channels and nightly in the release process". Suggest rewording to "see beta / experimental and nightly channels in the release process"

Comment (danielrozenberg) [reply contributing/DEVELOPING.md]: Done
